yunwoo/programmers-2/스킬트리.js

```diff
@@ -0,0 +1,24 @@
+function solution(skill, skill_trees) {
+  const reg = new RegExp(`[^${skill}]`, "g");
+  let count = 0;
+
+  skill_trees.forEach((target) => {
+    const convertedSkill = target.replace(reg, "");
+    let isOrder = true;
+
+    for (let i = 0; i < convertedSkill.length; i++) {
+      if (!(convertedSkill[i] === skill[i])) isOrder = false;
+    }
```

Comment:
```suggestion
    for (let i = 0; i < convertedSkill.length; i++) {
      if (convertedSkill[i] !== skill[i]) {
        isOrder = false;
      }
    }
```
- !(convertedSkill[i] === skill[i]) 이 아닌 convertedSkill[i] !== skill[i] 로 작성하실 수 있습니다

Reply:
- 아! 맞네요!! 감사합니다 🙇‍♂️